Library/Homebrew/cmd/search.rb

```diff
@@ -141,13 +141,20 @@ def search_tap(user, repo, rx)
     []
   end
 
-  def search_formulae(rx)
+  # @param [Regexp] rx
+  # @param [Hash] options
+  # @option options [Boolean] :installed If true, include installed formulae
```

Comment:
Maybe we can add an option(`:highlight_installed`), it can be used to turn off  highlight for non-stty cases.